services/tables/src/test/resources/cluster-test-properties.yaml

```diff
@@ -16,6 +16,11 @@ cluster:
         parameters:
           key2: value2
           token: xyz
+    storage-selector:
```

Comment:
we should make these properties `null` defaultable, ie, if storage-selector is not provided, it will select `DefaultStorageSelector`

Reply:
done